### PR TITLE
add windows test

### PR DESCRIPTION
**Pull request by ksimon1**
script which tests windows template in stdci.

### CODE REVIEW

**Comment by ksimon1** [comment]
ci test please

**Comment by ksimon1** [comment]
ci test please

**Comment by ksimon1** [comment]
ci test please

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by ksimon1** [comment]
ci test please

**Comment by MarSik** [comment]
ci add to whitelist

**Comment by ksimon1** [comment]
ci test please

**Comment by ksimon1** [comment]
ci test please

**Comment by ksimon1** [comment]
@MarSik @fabiand @fromanirh @yanirq please can you review?

File: automation/test.sh [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+_oc process -o json --local -f $windowsTemplatePath NAME=win2k1r2 PVCNAME=disk-windows | \
+jq '.items[0].spec.template.spec.volumes[0]+= {"ephemeral": {"persistentVolumeClaim": {"claimName": "disk-windows"}}} | 
+del(.items[0].spec.template.spec.volumes[0].persistentVolumeClaim)' | \
+_oc apply -f -
+
+curl -Lo virtctl \
+    https://github.com/kubevirt/kubevirt/releases/download/$VERSION/virtctl-$VERSION-linux-amd64
+  
+chmod +x virtctl
```
Comment (fromanirh) [inline automation/test.sh]: minor: I'd setup all the deps up front

File: automation/test.sh [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+      - "3600"
+    imagePullPolicy: Always
+    name: winrmcli
+restartPolicy: Always
+EOF
+
+# Make sure winrmcli pod is ready
+set +e
+current_time=0
+while ! _oc get pod winrmcli -o yaml | grep "phase: Running" ; do
```
Comment (fromanirh) [inline automation/test.sh]: I'm not fond of this kind of checks. Feels fragile. Maybe we can use -o json and jq?

**Comment by MarSik** [comment]
[(5)[presets-and-profiles] Functional tests for Windows template [initiative-always-ready-release]](https://trello.com/c/SPjqYB0J/41-5presets-and-profiles-functional-tests-for-windows-template-initiative-always-ready-release)

File: automation/test.sh (hunk 2 above)
Comment (ksimon1) [reply automation/test.sh]: updated

**Pull request merged**